docker/docker-entrypoint.sh

```diff
@@ -106,12 +110,6 @@ docker_process_init_files() {
 	done
 }
 
-start_server() {
```

Comment:
It actually started nothing because "$@" means empty string because we didn't pass arguments to start_server()